src/cov.jl

```diff
@@ -29,6 +29,21 @@ scattermat_zm(x::DenseMatrix, wv::WeightVec, vardim::Int) =
     _symmetrize!(Base.unscaled_covzm(x, _scalevars(x, values(wv), vardim), vardim))
 
 if VERSION < v"0.5.0-dev+679"
```

Comment:
Since there's a branch here, you'll have to duplicate the docs inside the other branch, or to specify only once before the branch with:
```julia
"""
docs
"""
function scattermat end


"""
docs
"""
cov
```